go-controller/pkg/node/gateway_init.go

```diff
@@ -184,6 +173,16 @@ func (n *OvnNode) initGateway(subnets []*net.IPNet, nodeAnnotator kube.Annotator
 			klog.Errorf("Unable to set primary IP net label on node, err: %v", err)
 		}
 	}
+	var loadBalancerHealthChecker *loadBalancerHealthChecker
+	var portClaimWatcher *portClaimWatcher
+
+	if config.Gateway.NodeportEnable {
+		loadBalancerHealthChecker = newLoadBalancerHealthChecker(n.name)
+		portClaimWatcher, err = newPortClaimWatcher(n.recorder, v4IfAddr, v6IfAddr)
```

Comment:
I"m not sure this commit is needed. It depends on the outcome of the comments in the first commit around handling node port to any IP on the host, vs specific node IP assigned in kubernetes.

Reply:
Why does this commit depend on that? We're binding to all addresses on the node, when we should only be binding to the node IP, right? Right now we could have issues with ephemeral ports used during connections and which could collide with the node port bound by ovnkube-node.

Reply:
Just restating what we discussed today for others: since we would use dst-type LOCAL it would affect any local IP on the host bound to the nodePort. This commit would reduce the port claim to just the node IP, so it doesn't make sense to do this when we go with dst-type local.